aiounifi/models/device.py

```diff
@@ -348,7 +348,7 @@ class TypedDevice(TypedDict):
     adopted: bool
     antenna_table: list[TypedDeviceAntennaTable]
     architecture: str
-    board_rev: int
+    adoption_completed: int
```

Comment:
You can use `NotRequired` for optional properties, I will add that to more items later on.
`from typing_extensions import NotRequired`
```suggestion
    adoption_completed: int
    board_rev: NotRequired[int]
```

Reply:
Thanks for the hint